six/modules/python/six.sicd/source/sicdUtils.py

```diff
@@ -39,3 +41,32 @@ def writeXML(complexData, pathname):
     finally:
         outputStream.close()
 
+
+def writeNITF(pathname, complexData, image, schemaPaths=VectorString()):
```

Comment:
Should we make these schema paths a Python list (and then convert to a `VectorString()` inside this function)?